apps/website/src/composables/wallet.ts

```diff
@@ -1,47 +1,69 @@
 import { ref } from 'vue'
 import { ethers } from 'ethers'
 import { WalletProvider } from '@/interfaces/WalletProvider'
+import useIopay from '@/composables/iopay'
+
+const { getIoPayAccounts, getIoPayProvider, sendIoPayTransaction } = useIopay()
 
 const defaultProviders = {
   metamask: undefined,
   coinbase: undefined,
+  iopay: undefined,
 }
 
 export default function useWallet() {
   const ethereum: any = window.ethereum
   const availableProviders = ref<Record<string, WalletProvider>>(
     getAvailableProviders(ethereum)
   )
-  const selectedProvider = ref<WalletProvider>({})
+  const selectedProvider = ref<WalletProvider | string>({}) // Adding string type to support 'iopay' throws new ts errors in sendTransaction
   const selectedAccount = ref<string>('')
-  const toAddress = ref<string>('') // Test to address: 0xD4e5faa8aD7d499Aa03BDDE2a3116E66bc8F8203
-  const amount = ref<string>('')
+  const amount = ref<string>('') // Use '1' to test
+  const toAddress = ref<string>('')
+  // Test ethereum send to address : 0xD4e5faa8aD7d499Aa03BDDE2a3116E66bc8F8203
+  // Test iotex send to address: acc://06da5e904240736b1e21ca6dbbd5f619860803af04ff3d54/acme
 
   async function connectWallet(provider: string) {
```

Comment:
Refactor to handle the strings mentioned above, rather than the provider/string mix.